src/index.css

```diff
@@ -49,3 +49,12 @@ main > .leaflet-container {
 ::-webkit-scrollbar-thumb:hover {
   background: hsla(81.9, 55.4%, 51.6%, 1);
 }
+
+/* Map styling */
+.leaflet-container {
+  background: #F0F0E8;
+}
+.leaflet-tile-pane {
+  opacity: 0.8;
```

Comment:
Can it be done without adjusting opacity? It may slow down map browsing, but ok if this is only way to give it a bluish tint from the background.

Reply:
I'm gonna remove opacity a bit later when I get home from work ;)

Reply:
Certainly not a blocker.